docs/USAGE.md

```diff
@@ -461,3 +462,13 @@ python manage.py streamdatamigration remove <app_label> <block_name> <path1> <pa
 
 where `<path>` would look like `<model_name>.<streamfield_name>....<parent_block_name>`
 
+## streamchangedetect
+
+A management command to detect changes made to streamfields from the last project state in 
```

Comment:
I think this deserves a higher profile! Could we add a little more about when you use it - before running `makemigrations` for your changes for example? Can we make it clear that it doesn't just detect changes, it generates the migrations too? Also, I think this is important enough to deserve adding to the readme